packages/react-core/src/components/Badge/__tests__/Badge.test.tsx

```diff
@@ -1,10 +1,46 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
 import { Badge } from '../Badge';
-import React from 'react';
-import { render } from '@testing-library/react';
 
-Object.values([true, false]).forEach(isRead => {
-  test(`${isRead} Badge`, () => {
-    const { asFragment } = render(<Badge isRead={isRead}>{isRead ? 'read' : 'unread'} Badge</Badge>);
-    expect(asFragment()).toMatchSnapshot();
-  });
+test('Renders without children', () => {
+  render(
+      <Badge data-testid="badge" />
+  );
+  expect(screen.getByTestId('badge')).toBeVisible();
```

Comment:
Can you adjust this a bit to match the pattern you see in the [Avatar tests](https://github.com/Dominik-Petrik/patternfly-react/blob/21905a3b40f45ce108c77d10d1e62d51082836a5/packages/react-core/src/components/Avatar/__tests__/Avatar.test.tsx#L6-L11)? Just for this first test.

The reasoning behind the linked approach is that it serves as a basic sanity check to make sure something renders, while not requiring that the component spreads props to the returned element.

Reply:
Thank you for the review! This should now be updated!